CharacterGenerator/CharacterGenerator/BenderDetailViewController.m

```diff
@@ -0,0 +1,64 @@
+//
+//  BenderDetailViewController.m
+//  CharacterGenerator
+//
+//  Created by Christian Maldonado on 8/4/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "BenderDetailViewController.h"
+
+@interface BenderDetailViewController ()
+@property (weak, nonatomic) IBOutlet UILabel *nameLabel;
+
+@property (weak, nonatomic) IBOutlet UIImageView *blownUpImageView;
+
+@property (weak, nonatomic) IBOutlet UILabel *color;
+@property (weak, nonatomic) IBOutlet UILabel *texture;
+@property (weak, nonatomic) IBOutlet UIImageView *animalImageView;
+@property (weak, nonatomic) IBOutlet UIImageView *locationImageView;
+
+@end
+
+@implementation BenderDetailViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.nameLabel.text = self.bender.name;
```

Comment:
Don't include `label` in your variable names. `characterName` would be a better choice.